### ISSUE TITLE
Spike Abstract for Design Workflow

### ISSUE THREAD

**Issue by marcelbertram** [porsche-design-system/porsche-design-system#6]
**Scope**  
Test Abstract as versioning tool for the internal UI Kit development of the global sketch library and as distribution possibility for sharing either through Abstract or including a seperate library in the Abstract workflow. Prototype a design workflow and test critical conflict paths.

**Out of Scope**  
Integration into the development workflow

**Findings Process**
- For distribution of the global library we wont use Abstract, because it has not proper pull request and approval strategy. Instead it is just for developing in parallel. The distribution should be handled as part of the GitHub repository and GitHub Pages deployment using the XML sketch linking

**Findings Commercial**
- Pricing/Pricing Model: 450$ / seat base price, discounts for 50 or 100 seats package
- Purchasing: Direct invoicing or via Sub-Vendor (tbd. Comparex, Megasoft)
- Infrastructure Location: Server in USA, security audit available
- Terms: Custom terms possible, e.g. location of jurdication in EU

**Subtasks**  
- [x] Setup Abstract trial organization and accounts
- [x] Setup Porsche UI Kit project including the finalized contents
- [x] Test using an external library integration into the Abstract project development
- [x] Test using an internal Abstract library beeing used in Abstract project development
- [x] Test user an permission management
- [x] Test workflow for changing project or library files till merging
- [x] Test conflict path: Working on the same symbol / artboard
- [x] Document the findings and a possible design workflow
- [x] Start research on possibilities, facts and cost of an enterprise account
